src/OrchardCore.Modules/Orchard.OpenId/README.md

```diff
@@ -103,3 +103,25 @@ OpenID Connect apps require the following configuration.
       "AllowHybridFlow": false
 }
 ```
+
+## Configuring Certificates
+
+### Windows / IIS
+
+Several tools are availble for generating a signing certificate on Windows and/or IIS, for example:
+
++ IIS Server Manager _(offers not much control)_
+    1. Server Certificates
+    2. Create Self-Signed Certificate 
++ PowerShell _(offers full control)_
+    1. `New-SelfSignedCertificate` https://technet.microsoft.com/en-us/itpro/powershell/windows/pkiclient/new-selfsignedcertificate
```

Comment:
This cmdlet is insanely obscure/complicated for beginners, so we should consider giving a concrete example. E.g:

```
New-SelfSignedCertificate `
    -Subject "connect.example.com" `
    -FriendlyName "Example.com Signing Certificate" `
    -CertStoreLocation "cert:\LocalMachine\My" `
    -KeySpec Signature `
    -KeyUsage DigitalSignature `
    -KeyUsageProperty Sign `
    -TextExtension @("2.5.29.37={text}1.3.6.1.5.5.7.3.1") `
    -KeyExportPolicy NonExportable `
    -KeyAlgorithm RSA `
    -KeyLength 4096 `
    -HashAlgorithm SHA256 `
    -NotAfter (Get-Date).AddYears(5) `
    -Provider "Microsoft Enhanced RSA and AES Cryptographic Provider"
```

This snippet must run as admin. It generates a 4096-bit signing certificate, stores it in the machine store and returns the thumbprint that must be set in the OpenID module settings.

(not tested, so please give it a try before using it as-is :sweat_smile:)